opencood/data_utils/datasets/intermediate_fusion_dataset.py

```diff
@@ -447,6 +447,8 @@ def get_pairwise_transformation(self, base_data_dict, max_cav):
                 for j in range(len(t_list)):
                     # identity matrix to self
                     if i == j:
+                        t_matrix = np.eye(4)
```

Comment:
Since you have implemented the project_later, maybe removing the warnings to avoid confusion for other users.

Reply:
Of course!